docs/maintainers/faqs.md

```diff
@@ -131,6 +131,22 @@ Short answer: no. For the moment, we have all the v3 tracks in this single monor
 
 The basic communication channel for maintainers is Slack, and in particular the maintainer-only `#maintaining-<track>` channel. For communication with everyone else, GitHub issues should be used, for obvious reasons (discoverability, familiarity, etc.). We've also enabled the GitHub Projects feature on the v3 repo, so you can organize your track's v3 transition there too.
 
+## Should I use citations
+
+Yes! Please use citations in any reference documentation (files under `**/reference/**`). We are using the [Science of Computer Programming citation style][socp-citation-style]. This means that when citing website, please use this format:
+
+> $TITLE, $SOURCE. ($PUBLISHED_DATE). $URL (accessed \$ACCESSED_DATE).
+
+For example:
+
+> [1] Actor model, Wikipedia. (2020). https://en.wikipedia.org/w/index.php?title=Actor_model&oldid=939106706 (accessed February 29, 2020).
+
+To include a citation in reference documentation, add the citation reference to the bottom of the page. Then add with a line with the `---` separator above the citation. Finally, put the cited text in [blockquotes][blockquotes] and append a superscript referring to the citation number (usually `<sup>1</sup>`). See the [actor model reference documentation][actor_model-reference-documentation] for an example.
```

Comment:
> Finally, put the cited text in [blockquotes][blockquotes] 

That is only true for direct quotes, not for documents that are based on or derived from other sources that you'd still want to cite, isn't it?

For example in the macro and metaprogramming files below, the quoted text is *not* a direct quote and shouldn't be marked as such.

Reply:
Good catch. I'll update.